### PR TITLE
Issue 158

### LINKED ISSUE #158: Check CSS for accessories tables
post import, Bryan says there are some CSS fixes needed for tables, especially in mobile.
example:
https://main--airlessco--hlxsites.hlx.page/na/en/products/accessories/revtipspraytips

### PR DESCRIPTION
**Pull request by helms-charity**
View in mobile for best results

Fix #158 

Test URLs:
- Before: https://main--airlessco--hlxsites.hlx.page/emea/es/products/accessories/revtipspraytips
- After: https://issue-158--airlessco--hlxsites.hlx.page/emea/es/products/accessories/revtipspraytips

### CODE REVIEW

**Comment by bstopp** [comment]
I'm having trouble figuring out which css change fixes which item in the referenced issue.

File: styles/styles.css [hunk 1]
```diff
@@ -102,6 +102,7 @@ h1 {
 h1 b, h1 strong {
   text-transform: uppercase;
   padding-right: 5px;
+  word-wrap: break-word;
```
Comment (bstopp) [inline styles/styles.css]: I'm not sure we should break words arbitrarily via CSS. The meaning of the word may change.

File: blocks/productcards/productcards.css [hunk 2]
```diff
@@ -57,7 +56,8 @@ h1#accessories-title strong {
 }
 
 @media screen and (min-width: 1024px) {
-  .productcards-container {
+  .card-heading.productcards-container {
```
Comment (bstopp) [inline blocks/productcards/productcards.css:59]: Why this change?

**Review by bstopp — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 1 above)
Comment (helms-charity) [reply styles/styles.css]: OK, good point.

File: blocks/productcards/productcards.css (hunk 2 above)
Comment (helms-charity) [reply blocks/productcards/productcards.css:59]: Because it matches the instance in line 12.

Comment (bstopp) [reply blocks/productcards/productcards.css:59]: i meant the whole change - why remove the width declaration above, and add it on this breakpoint?

Comment (helms-charity) [reply blocks/productcards/productcards.css:59]: See screenshot at https://github.com/hlxsites/airlessco/issues/158#issuecomment-1442452870

**Review by bstopp — APPROVED** [review]
